### PR TITLE
Single Dockerfile for bat-go

### PR DESCRIPTION
**Pull request by husobee**
### Summary

Provides one dockerfile with all bat-go executables contained.

### Type of change ( select one )

- [ ] Product feature
- [ ] Bug fix
- [ ] Performance improvement
- [ ] Refactor
- [x] Other

### Tested Environments

- [ ] Development
- [ ] Staging
- [ ] Production

### Before submitting this PR:

- [ ] Does your code build cleanly without any errors or warnings?
- [ ] Have you used auto closing keywords?
- [ ] Have you added tests for new functionality?
- [ ] Have validated query efficiency for new database queries?
- [ ] Have documented new functionality in README or in comments?
- [ ] Have you squashed all intermediate commits?
- [ ] Is there a clear title that explains what the PR does?
- [ ] Have you used intuitive function, variable and other naming?
- [ ] Have you requested security / privacy review if needed
- [ ] Have you performed a self review of this PR?

### CODE REVIEW

**Review by evq — COMMENTED** [review]
Looks good! We should probably update the `Makefile` as well - there's a ref there to the `Dockerfile.micro`

File: Makefile [hunk 1]
```diff
@@ -68,11 +68,6 @@ instrumented:
 	sed -i'bak' 's/client_duration_seconds/ratios_client_duration_seconds/g' utils/clients/ratios/instrumented_client.go
 	sed -i'bak' 's/client_duration_seconds/reputation_client_duration_seconds/g' utils/clients/reputation/instrumented_client.go
 
-microservice-docker:
-	docker build --build-arg COMMIT=$(GIT_COMMIT) --build-arg VERSION=$(GIT_VERSION) \
-		--build-arg BUILD_TIME=$(BUILD_TIME) -t bat_go-microservice:latest -f ./Dockerfile.micro .
-	docker tag bat_go-microservice:latest bat_go-microservice:$(GIT_VERSION)
-
 rewards-docker: microservice-docker
```
Comment (evq) [inline Makefile:71]: another ref here

Comment (husobee) [reply Makefile]: resolved here: https://github.com/brave-intl/bat-go/pull/524/commits/22a8a61669684e37e1d36f0e0121bac50eb1f353

improved the makefile to allow for sub docker image creation based on target specified.

**Review by hspencer77 — APPROVED** [review]
Solid.

**Pull request merged**